api/services/ocean/spark/parameters/query/searchTerm.yaml

```diff
@@ -0,0 +1,7 @@
+in: query
+name: searchTerm
+required: false
+schema:
+  type: string
+description: >
+  Optionally search on the application & job IDs and display names
```

Comment:
Optionally search for a substring of the application ID, application display name, job ID or job display name.

Reply:
Thanks!